css/components/services.css

```diff
@@ -1,3 +1,8 @@
 .service-icon {
   font-size: 35px;
 }
+
+.services-selector:hover {
```

Comment:
norėtųsi dar to transition, ir matau kad h2 ir p tagu hoverio transition laikai turetu skirtis. visai graziai atrodo.
paziurejus dizaine ikona pirmiau nusispalvina tada h2

Reply:
as maciau skirtuma, kazkaip man atrode siek tiek geriau kad abu vienu metu pasijungia. Bet galiu pakeisti no problem :)

Reply:
pritariu siulymui pakeisti transition laika, kad butu skirtingas h2 ir p tagu, bei patarciau transition savybe deti ne ant hoverio, o po atskira ant h2 ir p. hoveris pasigaus savybe pats.

Reply:
jeigu sudedu transition ant atsikuru el, lieka tik vienas active hover metu. Nezinau kaip padaryti atsikai :(

Reply:
services-selector:hover > h2,
services-selector:hover > p {
color: bla bla bla;
} 
gal suveiks, pabandyk ir dar atskirai ant h2 ir p ne hoverio metu po transitiona uzdek :)

Reply:
gal dar tiems h2 ir p po atskira klase reiktu pasikurt, kad hoverio metu visu puslapyje h2 ir p neuzspalvintu netycia

Reply:
Aciu, atrodo pataisiau :)

Reply:
ant ikonu gradienta uzdejau, taip gal panasiau i originala:
<img width="1440" alt="Screen Shot 2022-04-26 at 1 41 00 PM" src="https://user-images.githubusercontent.com/80914644/165282720-50f1d06b-bee3-4955-b630-56533c337b77.png">
su transition liuks!

Reply:
ok